proc_echo_mw.py

```diff
@@ -14,10 +14,11 @@
 # to use: as a rule of thumb, make the white boxes
 # about 2x as far as it looks like they should be
 # leave this as a loop, so you can load multiple files
-for searchstr,exp_type,nodename,postproc,freq_range,time_range in [
-        ["200306_DNP_lg_probe_w34.*", 'test_equip', 'signal',
-            'spincore_ODNP_v1', (-300,300), (None,0.05)]
+for searchstr,exp_type,nodename,postproc,freq_range,max_t in [
+        ["201124_4AT100uM_DNP_cap_probe_1", 'ODNP_NMR_comp', 'signal',
+            'spincore_ODNP_v1', (-500,500), 0.065]
```

Comment:
The previous parameters allow for the case where there's an echo that's not short time -- where there is noise between the start of acquisition and the rise of the echo.  Relatively inexpensive to permit this.

Reply:
Okay so my understanding of this was to go back to using the t_range instead of max_t

Reply:
yes, but above code doesn't show as outdated

Reply:
sorry realized I hadn't pushed!